src/EnterMediaDbClient.php

```diff
@@ -173,17 +173,27 @@ public function login() {
   /**
    * {@inheritdoc}
    */
-  public function upload(EmbridgeAssetEntityInterface $asset) {
+  public function upload(EmbridgeAssetEntityInterface $asset, array $metadata) {
```

Comment:
$metadata should default to an empty array to keep existing functionality for other locations upload() is used.